src/components/operationalStatus/OperationalStatus.tsx

```diff
@@ -43,10 +45,44 @@ export const OperationalStatus: FunctionComponent<OperationalStatusProps> = ({
   maintenances = null,
   filter,
 }) => {
+  type SortKey = keyof Service;
+  type SortDirection = "ascending" | "descending";
+  const [sort, setSort] = useState<{ key: SortKey; direction: SortDirection }>({
+    key: "name",
+    direction: "ascending",
+  });
+
+  const requestSort = (key: SortKey) => {
+    let direction = "ascending" as SortDirection;
+    if (sort.key === key && sort.direction === "ascending") {
+      direction = "descending";
+    }
+    setSort({ key, direction });
+  };
+
   const filteredData = filter
     ? services.filter((service) => service.status === filter)
     : services;
 
+  const sortedData = [...filteredData].sort((a, b) => {
+    let aValue = a[sort.key];
+    let bValue = b[sort.key];
+    if (!aValue || !bValue) return 0;
```

Comment:
This breaks sorting when 1 value is missing, eg. two services with failure, but only one has an `impact`